test/config_cat_test.exs

```diff
@@ -250,13 +266,18 @@ defmodule ConfigCatTest do
           fetch_policy: FetchPolicy.lazy(cache_expiry_seconds: 300)
         )
 
+      call_times = 10
+
       APIMock
-      |> expect(:get, 1, fn _url, _headers ->
+      |> expect(:get, 1, fn _url, _headers, _options ->
         {:ok, %Response{status_code: 200, body: config}}
       end)
 
-      ConfigCat.get_value(feature, "default", client: client)
-      assert ConfigCat.get_value(feature, "default", client: client) == value
+      result = Enum.reduce 1..call_times, 0, fn _n, _a ->
+        ConfigCat.get_value(feature, "default", client: client)
+      end
```

Comment:
What is this change about? Doesn't seem related to the proxy option, and I'm curious about what might be different between calling the function once vs 10 times.

Reply:
It's unrelated. It was just a refactoring, something I stumbled that I thought it was weird. The code was like this:
```
ConfigCat.get_value(feature, "default", client: client)
ConfigCat.get_value(feature, "default", client: client)
```

At first, I thought it was a mistake. Then I removed one line and the tests failed... then I understood what it was about and I tried to make it more explicit of what it is about.

Calling 1 or 10 times, in theory, doesn't make  any difference because the call is cached but I changed back to 2 👍